client/common/types/Task.ts

```diff
@@ -0,0 +1,14 @@
+export default interface Task {
+  id: Number;
+  title: String;
+  description: String;
+  status: String;
+  priority: String;
+  tags: String[]; //Tags[]
+  deadline: String;
+  milestone_id: String;
```

Comment:
all of the ids are stored as Int type, or Number in the database.

Reply:
Thank you, i'lll change that right away